src/sessions/video-session-handler.ts

```diff
@@ -626,4 +646,30 @@ export default class VideoSessionHandler extends BaseSessionHandler {
     element.srcObject = createNewStreamWithTrack(track);
     return element;
   }
+
+  /**
+   * Parse the trackId from a passed in SDP for a given media type
+   *
+   * SDP will look like:
+   * ```
+   * // global stuff...
+   * m=audio 1 UDP/TLS/RTP/SAVPF 96
+   * // info about the audio offer...
+   * a=msid:cbf2ec37-5e50-4ac4-9ae7-1d1dc4508071 19d58781-f708-4945-be91-2758052273bd
+   * m=video 1 UDP/TLS/RTP/SAVPF 97 98
+   * // info about the video offer...
+   * a=msid:cbf2ec37-5e50-4ac4-9ae7-1d1dc4508071 1e3d9e8b-d407-47ee-8dcf-6b5912889a28
+   * ```
+   *
+   * `m=` acts as the delimiter for each audio/video track in the offer
+   * `a=misd:{ID for the media stream} {ID for the media track (this is what we will look for)}
+   *
+   * @param sdp to parse
+   * @param kind media type to look for
+   */
+  getTrackIdFromSdp (sdp: string, kind: 'video' | 'audio'): string {
```

Comment:
do we know why getting the trackId from the receiver doesn't work for ff?

Reply:
I answered in standup, but I will answer here too for record sake. Firefox's media stream/track Ids look like `{some-hash}` where the `{}` are part of the id. Most other technologies (including MMS) just uses a hash with no brackets. This is causing the `pc.getReceivers()`'s trackIds to not match the `participants[].tracks[].sinks` Ids.